inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/dirty/DirtySinkHelper.java

```diff
@@ -95,6 +95,9 @@ public void invoke(T dirtyData, DirtyType dirtyType, Throwable e) {
                         .setLogTag(dirtyOptions.getLogTag())
                         .setDirtyMessage(e.getMessage())
                         .setIdentifier(dirtyOptions.getIdentifier());
+                LOGGER.info("dirty sink build debug: {}, {}, {}, {}, {}, {}",
```

Comment:
log.info ("debug")? maybe log.debug or log.info( "info")

Reply:
We can't get the exact meaning of each element printed. 
Suggestion :
LOGGER.info("dirty sink build  data: {}, type: {}, lables: {}, tag: {}, Identifier: {}, errorMsg: {}", dirtyData, dirtyType, dirtyOptions.getLabels(), dirtyOptions.getLogTag(), dirtyOptions.getIdentifier(), e.getMessage());